src/test/java/picard/illumina/IlluminaBasecallsToSamTest.java

```diff
@@ -337,22 +352,28 @@ private void runStandardTest(final int lane, final String jobName, final String
             final String[] fields = line.split("\t");
             final File outputSam = new File(outputDir, StringUtil.join("", Arrays.copyOfRange(fields, 0, concatNColumnFields)) + ".sam");
             outputSam.deleteOnExit();
-            samFiles.add(outputSam);
+            samFiles.add(new File(outputSam.getParentFile(), tilePrefix + outputSam.getName()));
             writer.println(line + "\t" + outputSam);
         }
         writer.close();
         reader.close();
 
-        Assert.assertEquals(runPicardCommandLine(new String[]{
-                "BASECALLS_DIR=" + baseCallsDir,
-                "LANE=" + lane,
-                "RUN_BARCODE=HiMom",
-                "READ_STRUCTURE=" + readStructure,
-                "LIBRARY_PARAMS=" + libraryParams
-        }), 0);
+        List<String> args = new ArrayList<>();
+        args.add("BASECALLS_DIR=" + baseCallsDir);
+        args.add("LANE=" + lane);
+        args.add("RUN_BARCODE=HiMom");
+        args.add("READ_STRUCTURE=" + readStructure);
+        args.add("LIBRARY_PARAMS=" + libraryParams);
+        if(tile != null){
+            args.add("PROCESS_SINGLE_TILE=" + tile);
+        }
+
+
+        Assert.assertEquals(runPicardCommandLine(args), 0);
 
         for (final File outputSam : samFiles) {
             IOUtil.assertFilesEqual(outputSam, new File(testDataDir, outputSam.getName()));
+            outputSam.deleteOnExit();
```

Comment:
Not sure what this is for. The outputDir will be deleted below so this doesn't need to be done. If the idea is that we want to delete files even if the test fails, it would be better to just delete the file here instead of using `deleteOnExit()`.